package.json

```diff
@@ -6,15 +6,11 @@
     "hatchetapp.net",
     "www.hatchetapp.net"
   ],
-  "scripts": {
-    "start": "server.js"
-  },
   "engines": {
     "node": "0.8.x"
   },
   "dependencies": {
-    "express": "2.5.10",
-    "express-cdn": "0.0.3",
+    "express": "4.17.1",
```

Comment:
This is a major API change for express. Bumping the package version here without other code changes will break the entire application. This kind of major change should be made upstream and not in this fork.